test/python/circuit/test_controlled_gate.py

```diff
@@ -488,7 +489,7 @@ def test_multi_controlled_u1_matrix(self, num_controls):
                     qc.x(q_controls[idx])
 
             backend = BasicAer.get_backend("unitary_simulator")
-            simulated = execute(qc, backend).result().get_unitary(qc)
+            simulated = backend.run(transpile(qc, backend)).result().get_unitary(qc)
```

Comment:
I wouldn't mind just changing these into `simulated = Operator(qc)`, but this is also fine.

Reply:
done in 6ef382aab412f744a330b36fbc8eda6bc623082c